rtil/src/native/map_editor.rs

```diff
@@ -2,7 +2,7 @@ use std::cell::Cell;
 use std::fmt::{Formatter, Pointer};
 use std::ops::Deref;
 use std::sync::Mutex;
-use crate::native::{ArrayWrapper, ObjectIndex, StructValueWrapper, UeObjectWrapperType, UeScope, UObject};
+use crate::native::{ArrayWrapper, ClassWrapper, ObjectIndex, ObjectWrapper, StructValueWrapper, UeObjectWrapperType, UeScope, UObject};
```

Comment:
Are those actually needed? The compiler should likely warn about them. Did you check that you didn't introduce any new warnings?